main.go

```diff
@@ -52,18 +53,18 @@ func main() {
 	}
 
 	log.Println("Getting organization members...")
-	members, err := processor.Members(cfg.OrgName)
+	members, err := processor.Members(cfg.OrgName, cfg.HideTeamMembers)
 	if err != nil {
 		log.Fatalf("Error getting members: %v", err)
 	}
 
 	log.Println("Getting organization teams...")
-	teams, parents, err := processor.Teams(cfg.OrgName, orgID)
+	teams, parents, err := processor.Teams(cfg.OrgName, cfg.HideTeamMembers, orgID)
```

Comment:
To minimize blast radius, and not break tests, I suggest wrapping calls to `processor.Memebers` and `FindMembersWithoutTeam` in condition here, like:

```go
	log.Println("Getting organization teams...")
	teams, parents, err := processor.Teams(cfg.OrgName, cfg.HideMembers, orgID)
	if err != nil {
		log.Fatalf("Error getting teams: %v", err)
	}

	var members []string
	if !cfg.HideMembers {
		log.Println("Getting organization members...")
		members, err := processor.Members(cfg.OrgName)
		if err != nil {
			log.Fatalf("Error getting members: %v", err)
		}

		membersWitoutTeam := FindMembersWithoutTeam(teams, members)
		if len(membersWitoutTeam) > 0 {
			teams["NO_TEAM"] = membersWitoutTeam
		}
	}
```